scheduler/test/cook/test/scheduler/constraints.clj

```diff
@@ -22,13 +22,13 @@
             [cook.scheduler.constraints :as constraints]
             [cook.scheduler.data-locality :as dl]
             [cook.scheduler.scheduler :as sched]
-            [cook.test.testutil :refer [create-dummy-group create-dummy-instance create-dummy-job create-dummy-job-with-instances
+            [cook.test.testutil :refer [create-dummy-group create-dummy-instance create-dummy-job create-dummy-job-with-instances create-pool
                                          restore-fresh-database! setup]]
             [cook.tools :as util]
-            [datomic.api :as d :refer [db]])
-  (:import (java.util Date UUID)
-           (org.joda.time DateTime)
-           (org.mockito Mockito)))
+            [datomic.api :as d :refer (db)])
+  (:import [java.util Date UUID]
+           org.joda.time.DateTime
+           org.mockito.Mockito))
```

Comment:
Were the changes here intentional?

Reply:
no :( changed back

Reply:
I think the `:refer (db)` one is still there.